src/pytest_cov/engine.py

```diff
@@ -450,6 +450,7 @@ def start(self):
             data_suffix=_data_suffix(f'w{self.nodeid}'),
             config_file=self.cov_config,
         )
+        self.cov._warn_unimported_source = False
```

Comment:
This should probably include all those other hacks, after all:
```suggestion
        self.cov._warn_no_data = False
        self.cov._warn_unimported_source = False
        self.cov._warn_preimported_source = False
```

Reply:
Yeah, I only know that only one has a visible effect of suppressing a false negative in ansible-pylibssh. I've been trying to think of a reliable test but no luck so far.